package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "openboxes-e2e",
+  "version": "1.0.0",
+  "description": "End to end testing for OpenBoxes",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "lint-fix": "eslint . --fix",
+    "format": "npx prettier --write .",
+    "test": "npx playwright test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openboxes/openboxes-e2e.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/openboxes/openboxes-e2e/issues"
+  },
+  "homepage": "https://github.com/openboxes/openboxes-e2e#readme",
+  "devDependencies": {
+    "@playwright/test": "1.42.1",
+    "@types/node": "20.11.30",
+    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "eslint": "8.57.0",
+    "eslint-config-prettier": "9.1.0",
+    "eslint-config-standard-with-typescript": "37.0.0",
+    "eslint-plugin-playwright": "1.0.1",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
```

Comment:
remove `^`

Reply:
Perhaps we could have a `~` for allowing patch releases?